README.md

```diff
@@ -1,6 +1,8 @@
 # ansible-role-banner #
 
-[![Build Status](https://travis-ci.com/cisagov/ansible-role-banner.svg?branch=develop)](https://travis-ci.com/cisagov/ansible-role-banner)
+[![GitHub Build Status](https://github.com/cisagov/ansible-role-banner/workflows/build/badge.svg)](https://github.com/cisagov/ansible-role-banner/actions)
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/cisagov/ansible-role-banner.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/ansible-role-banner/alerts/)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/ansible-role-banner.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/ansible-role-banner/context:python)
 
 An Ansible role for installing the NCATS login banner.
```

Comment:
There is a mention of NCATS in here that we should probably change to CISA.

Reply:
Resolved in 039bca1.